### PR TITLE
Improve build and fixes

### PR DESCRIPTION
**Pull request by NathanBnm**
@rickybas  I just improved the way your app is build through meson.

I changed the way how icons are installed too in order to follow the  [elementary HIG](https://elementary.io/docs/human-interface-guidelines#size). You have to install 16x16, 24x24, and 32x32 icons too.

Translation could not work so I made them work.

I added a `.editorconfig` as recommended by elementary.

Let me know if you have any questions.
I would be happy to add French translation once you'll have merged this.

### CODE REVIEW

File: src/MainWindow.vala [hunk 1]
```diff
@@ -54,7 +54,7 @@ public class MainWindow : Gtk.Dialog {
         Object (application: application,
                 icon_name: "com.github.rickybas.date-countdown",
                 resizable: false,
-                title: "Date Countdown");
+                title: (_("Date Countdown")));
```
Comment (ryonakano) [inline src/MainWindow.vala]:
```suggestion
                title: _("Date Countdown"));
```

I guess there is an extra couple of brackets.

Comment (NathanBnm) [reply src/MainWindow.vala]: Sure!

**Comment by rickybas** [comment]
Thanks for the contribution @NathanBnm !

**Comment by rickybassom** [comment]
Thanks for the contribution @NathanBnm !

Apologies for the delay.

**Pull request merged**